antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/atn/PredictionContext.kt

```diff
@@ -56,30 +58,33 @@ public abstract class PredictionContext protected constructor(
     /**
      * Convert a [RuleContext] tree to a [PredictionContext] graph.
      *
-     * Return [EmptyPredictionContext.Instance] if [outerContext] is empty or `null`.
+     * Return [EmptyPredictionContext] if [outerContext] is empty or `null`.
      */
+    @JvmStatic
     public fun fromRuleContext(atn: ATN, outerContext: RuleContext?): PredictionContext {
       val tempOuterContext = outerContext ?: ParserRuleContext.EMPTY
 
       // If we are in RuleContext of start rule, s, then PredictionContext
       // is EMPTY. Nobody called us. (if we are empty, return empty)
-      if (tempOuterContext.parent == null || tempOuterContext === ParserRuleContext.EMPTY) {
-        return EmptyPredictionContext.Instance
+      if (tempOuterContext.getParent() == null || tempOuterContext === ParserRuleContext.EMPTY) {
```

Comment:
Wouldn't `.parent` be interpreted as `getParent()`?

Reply:
@ftomassetti nope, because now the `protected var parent` field is marked with `@JvmField`.

Reply:
I see